custom_components/wundasmart/sensor.py

```diff
@@ -259,27 +264,30 @@ async def async_setup_entry(
     sensors = itertools.chain(
         (
             Sensor(wunda_id,
-               room["name"] + " " + desc.name,
+               desc.name if coordinator._separate_room_devices else room["name"] + " " + desc.name,
                coordinator,
-               desc)
+               desc,
+               room_id=wunda_id)

```

Comment:
Why is room_id set to wunda_id here and not the room id?

EDIT: Ah, I see, they resolve to being the same thing for devices for type ROOM